test/performance/tests/test_performance.py

```diff
@@ -30,3 +31,91 @@ def test_load_test(instances: List[harness.Instance]):
     assert "control-plane" in util.get_local_node_status(cluster_node)
     assert "control-plane" in util.get_local_node_status(joining_node)
     assert "control-plane" in util.get_local_node_status(joining_node_2)
+
+    configure_kube_burner(cluster_node)
+    process_dict = collect_metrics(instances)
+    run_kube_burner(cluster_node)
+    stop_metrics(instances, process_dict)
+    pull_metrics(instances)
+
+
+def test_single_node_load(session_instance: harness.Instance):
+    """Test the performance of a single node cluster with all features enabled."""
+    configure_kube_burner(session_instance)
+    process_dict = collect_metrics([session_instance])
+    run_kube_burner(session_instance)
```

Comment:
Will the kube-burner exits only when it's done pressure testing? Or will it start and continue in the background?
I'm asking this because I want to make sure we're not killing the metrics processes when the bench is still ongoing.

Reply:
Great question! The kube-burner command runs and blocks whereas we start the metrics subprocess in the background and kill it after the load test is over.